cmd/release-controller/http.go

```diff
@@ -959,7 +959,11 @@ func (c *Controller) httpReleases(w http.ResponseWriter, req *http.Request) {
 
 	authMessage := ""
 	if len(c.authenticationMessage) > 0 {
-		authMessage = fmt.Sprintf("<p>%s</p>", c.authenticationMessage)
+		if message, err := strconv.Unquote(c.authenticationMessage); err == nil {
```

Comment:
Why does authenticationMessage have surrounding quotes?  Can you provide the example input that leads to this?

Reply:
/close

I fixed this at the source: https://github.com/openshift/release/pull/20826